addons/mrp/tests/test_manual_consumption.py

```diff
@@ -230,3 +230,21 @@ def get_moves(mo):
         move_auto, move_manual = get_moves(backorder)
         self.assertEqual(move_auto.manual_consumption, False)
         self.assertEqual(move_manual.manual_consumption, True)
+
+    def test_update_manual_consumption(self):
+        """
+        Check that the manual consumption is set to true when the quantity is manualy set.
+        """
+        mo_form = Form(self.env['mrp.production'])
+        mo_form.product_id = self.bom_3.product_id
```

Comment:
It's a bit weird to use the product of bom 3 while you're using bom 1
Also, setting bom 1 on the form will define the finished product
```suggestion
```

Reply:
My bad ! Good catch :)